english/zoo/zoo.md

```diff
@@ -4,4 +4,7 @@ nature. I saw a lot of tigers, elephants, deer and a whole lot of other [animals
 I was pretty thirsty. There is a lemonade stand over there.
 I head over and buy some [lemonade](../drink/lemonade/lemonade.md)
 
-I choose to get better at [English](../get-better-at-english/english.md)
\ No newline at end of file
+
+I choose to get better at [English](../get-better-at-english/english.md)
+
+After my self trolling is complete, I decide to drink the lemonade.  As I lift the lemonade to my mouth I see the tigers have broken loose. [tiger-attack](../tiger-attack/tiger-attack.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.